pkg/infra/log/logtest/fake.go

```diff
@@ -0,0 +1,50 @@
+package logtest
+
+import (
+	"github.com/grafana/grafana/pkg/infra/log"
+)
+
```

Comment:
Nit. In the context of logtest maybe make more sense to name this just Logger?

Reply:
Could be, but we might want other kinds of test doubles, like mocks or whatnot.